README.md

```diff
@@ -45,16 +45,20 @@ If you find a bug, have a feature request, or want to contribute to Acts, please
 
 # <a name="license-authors">License and authors</a>
 
-This project is published under the Mozilla Public License, v. 2.0. Details of
-this license can be found in the [LICENSE](LICENSE) file or at
-http://mozilla.org/MPL/2.0/.
+This project is published under the terms of the Mozilla Public License, v. 2.0.
+A copy of the license can be found in the [LICENSE](LICENSE) file or at
+http://mozilla.org/MPL/2.0/ .
 
-Contributors to the Acts project are listed in [AUTHORS](AUTHORS).
+Contributors to the Acts project are listed in the [AUTHORS](AUTHORS.md) file.
 
-The Acts project contains a copy of [gcovr](http://gcovr.com) licensed under
-the 3-Clause BSD license.
+The Acts project is based on the ATLAS tracking software. A list of contributors
+to the ATLAS tracking repository can be found at
+http://acts.web.cern.ch/ACTS/ATLAS_authors.html .
 
-This software contains a copy of the `JSON for Modern C++`-library by Niels Lohmann licensed under the MIT License.
+The Acts project contains copies of the following external packages:
 
-The Acts project is based on the ATLAS tracking software. A list of contributors
-to the ATLAS tracking repository can be found <a href="http://acts.web.cern.ch/ACTS/ATLAS_authors.html">here</a>.
+*   [dfelibs](https:://gitlab.cern.ch/msmk/dfelibs) by Moritz Kiehn licensed
+    under the MIT license.
+*   [gcovr](http://gcovr.com) licensed under the 3-Clause BSD license.
+*   [JSON for Modern C++](https://github.com/nlohmann/json) by Niels Lohmann
+    licensed under the MIT License.
```

Comment:
I think this isn't true anymore. We don't ship `nlohmann::json` anymore (CMake downloads it), and `dfelibs` is a submodule. The bundled `gcovr` is not used anymore, so we can throw it out if it's still there.

Can you remove the entire section?

Reply:
Nevermind, we changed that.

Reply:
I'm still reaaaally uncomfortable with having these commited inside our repository thought :/

Reply:
Both `nlohmann::json` and `dfelibs` are shipped as source copies in `thirdparty` in order to not have submodules anymore (that is one of the reason for going towards a monorepo). We decided in https://gitlab.cern.ch/acts/acts-core/-/merge_requests/733 to ship `nlohmann::json` by copy for various reasons.